src/databricks/sqlalchemy/_parse.py

```diff
@@ -282,6 +282,7 @@ def get_pk_strings_from_dte_output(
 GET_COLUMNS_TYPE_MAP = {
     "boolean": sqlalchemy.types.Boolean,
     "smallint": sqlalchemy.types.SmallInteger,
+    "tinyint": sqlalchemy.types.SmallInteger,
```

Comment:
This should be  `type_overrides.TINYINT`, not `SmallInteger`, no?

Reply:
I do not know enough of the code to make the judgement call. I used my best guess to figure out the quickest solution that could work.

Reply:
Understood. Databricks has both `SMALLINT` and `TINYINT` types. The code you're editing has to do with column reflection of existing tables. These are mapped to SqlAlchemy types named `SmallInteger` (which is built-in to SQLAlchemy) and `TINYINT` (which is a custom type implemented in our dialect). In Python, a `TINYINT` in our dialect behaves _exactly_ the same as a `SmallInteger` (Python doesn't have any size restrictions for integers). That's why your version of the code _works for your use case_. But we should reflect the type properly back as `type_overrides.TINYINT. Otherwise SQLAlchemy will not properly reflect back the true schema of the table, which will be an issue for people using SQLAlchemy/Alembic to programmatically manage their table schemas.